src/clzones.cpp

```diff
@@ -244,6 +272,44 @@ plot_options::query_seed_result plot_options::query_seed()
     }
 }
 
+bool loot_options::query_at_creation()
+{
+    return query_loot() != canceled;
+}
+
+bool loot_options::query()
+{
+    return query_loot() == changed;
+}
+
+std::string loot_options::get_zone_name_suggestion() const
+{
+    if( !mark.empty() ) {
+        return string_format( _( "Loot: Custom : %s" ), mark );
+    }
+    return _( "Loot: Custom : No Filter" );
+}
+
+std::vector<std::pair<std::string, std::string>> loot_options::get_descriptions() const
+{
+    std::vector<std::pair<std::string, std::string>> options =
+                std::vector<std::pair<std::string, std::string>>();
```

Comment:
You do not need to call the default constructor explicitly. Just `std::vector<std::pair<std::string, std::string>> options;` is fine.